test/integration/new-storage/DeleteExpiredCmd.test.js

```diff
@@ -0,0 +1,119 @@
+const cassandra = require('cassandra-driver')
+const { TimeUuid } = require('cassandra-driver').types
+
+jest.mock('../../../src/helpers/validateConfig')
+const validateConfig = require('../../../src/helpers/validateConfig')
+const DeleteExpiredCmd = require('../../../src/new-storage/DeleteExpiredCmd')
+const { startBrokerNewSchema, createClient } = require('../../utils')
+
+validateConfig.mockImplementation(() => true)
```

Comment:
Hmm.. maybe better to use real `validateConfig`?

Reply:
In this case I need full config, I decided to have simple config https://github.com/streamr-dev/broker/pull/181/files#diff-dcc1117f055e924819d91caca7bfb141R103-R114